README.md

```diff
@@ -987,6 +988,19 @@ Flags:
 - :whale: `-q, --quiet`: Only display snapshots IDs
 - :whale: `--format`: Format the output using the given Go template, e.g, `{{json .}}`
 
+### :whale: nerdctl image prune
+Remove unused images.
+
+NOTE: The default behavior of this command is to remove all unused images.
+Inconsistent with docker image prune command, which by default removes only dangling iamge unless the --all flag is specified.
```

Comment:
Better to require `--all` then?

When `--all` is not specified, probably this command should do NOP with a warning message

Reply:
SGTM